etl/steps/data/garden/insee/2024-04-25/insee_premiere_1875.meta.yml

```diff
@@ -0,0 +1,174 @@
+# NOTE: To learn more about the fields, hover over their names.
```

Comment:
i believe some indicators are missing `processing_level`

Reply:
It's here in common. Maybe you wanted to write this in another yaml